backend/libexecution/liblist.ml

```diff
@@ -379,6 +379,28 @@ let fns : fn list =
               fail args)
     ; preview_safety = Safe
     ; deprecated = false }
+  ; { prefix_names = ["List::interleave"]
+    ; infix_names = []
+    ; parameters = [par "as" TList; par "bs" TList]
+    ; return_type = TList
+    ; description =
+        "Returns a new list with the first value from `as` then the first value from `bs`, then the second value from `as` then the second value from `bs`, etc, until one list ends, then the remaining items from the other list."
```

Comment:
Can you use the docstring format from here? https://github.com/darklang/dark/blob/main/docs/writing-docstrings.md